freemile2ru/document-manager
# merge development branch to master

## What does this PR do?

This PR merge the final working app to the master branch

## Description of Task to be completed?
built the app from start to finish following the stories created here [pivital tracker](https://www.pivotaltracker.com/n/projects/1991935)

## How should this be manually tested?
by 
1) clone the repo
2) inspect the package.json file and see the changes
3) run npm start to start the app and you will see the landing page
or checkout app on [heroku](https://freemiledocman.herokuapp.com/)

## Any background context you want to provide?
N/A

## What are the relevant pivotal tracker stories?
N/A

screenshot (if applicable)
N/A
